lib/modules/datasource/go/base.ts

```diff
@@ -73,7 +73,7 @@ export class BaseGoDatasource {
     goModule: string
   ): DataSource | null {
     const sourceMatch = regEx(
-      `<meta\\s+name="go-source"\\s+content="([^\\s]+)\\s+([^\\s]+)`
+      `<meta\\s+name="?go-source"?\\s+content="([^\\s]+)\\s+([^\\s]+)`
```

Comment:
this is strange, as this violates html standards, so this is a workaround for misbehaving API. 😕

Reply:
:shrug: